Astronomy/Astronomy/Photo.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Photo.swift
+//  Astronomy
+//
+//  Created by Karen Rodriguez on 5/18/20.
+//  Copyright © 2020 Wyatt Harrell. All rights reserved.
+//
+
+import Foundation
+
+@objc class Photo: NSObject {
+    @objc let sol: Int
+    @objc let cameraName: String
+    @objc let cameraFullName: String
+    @objc let photoDate: Date
+    @objc let imgSrc: URL
+    @objc let roverID: Int
+    @objc let roverName: String
+
+    @objc init(sol: Int, cameraName: String, cameraFullName: String, photoDate: Date, imgSrc: URL, roverID: Int, roverName: String) {
```

Comment:
You can also add `@objcMembers` before class Photo... and avoid having to add `@objc` to all of the class members. You might want to have more fine control over what's exposed to Objective-C, but I find `@objcMembers` just makes things a lot cleaner to read at least, and usually doesn't have any downsides.